route53_static.tf

```diff
@@ -182,3 +182,14 @@ resource "aws_route53_record" "dkim_CNAME" {
   type    = "CNAME"
   zone_id = aws_route53_zone.cyber_dhs_gov.zone_id
 }
+
+# ------------------------------------------------------------------------------
+# Set up mail.${aws_route53_zone.cyber_dhs_gov.name} as SES MAIL FROM
+# resource.
+# ------------------------------------------------------------------------------
+resource "aws_ses_domain_mail_from" "example" {
```

Comment:
I don't understand why "example" is being used as the name for this resource.

Reply:
Ah, that's because I copied the code from the TF docs.  See commit 865e81b.

Reply:
Well, as long as I'm not crazy. Thanks for fixing it 👍